### ISSUE TITLE
Port to Next.js

### ISSUE THREAD

**Issue by bfabio** [italia/developers.italia.it#1007]
Jekyll is getting limiting and as a consequence we are accumulating technical debt because of it. This won't get better as the site grows in functionality.

We should move to something more powerful and featureful on the frontend side, like Next.js ~~or Gatsby~~ **edit: we decided on Next.js.**

# General approach

We want to:
* port preserving the features / behaviour we have now
* do it in an incremental way, instead of a single final release
* do it section by section, so that ported sections live together with the old sections
* do it in a user-transparent way: the old and the new sections should not be noticeable by domain or different layout
* bonus: implement or fix current issue along the way, if it's reasonable to do it

## Preparation work
* [x] #1473
* [x] #1474

## Sections

We can look at the site as sections, with common characteristics, loosely ordered by effort required:

* [x] #1465
* [x] #1466
* [x] #1469
* [x] #1467
* [x] #1468
* [x] #1470
* [x] #1471
* [x] #1475
* [x] #1472

## Misc
* [x] #1476

**Comment by bfabio** [comment]
Started https://github.com/italia/next.developers.italia.it, which will implement the new Interoperability section initially.

**Comment by smonolo** [comment]
@bfabio Thanks for setting up the issues and a very basic roadmap. I guess we'll have a discussion on the individual issues when it comes to figuring out the technical implementation.

**Comment by bfabio** [comment]
@smonolo yes, I'm going to add details to the issues, for now the empty ones act as placeholders

**Comment by astagi** [comment]
Hi @smonolo @bfabio unfortunately atm the Next.js port is not planned. We're discussing on porting Developers Italia to other platforms (Astro.js is a strong candidate 😎).
